src/pages/ItemList.js

```diff
@@ -58,54 +58,89 @@ function ItemList({ token }) {
      * Return early if the checkbox was already checked
      * because we don't want to purchase an item twice.
      */
+
+    const itemId = docs.find((doc) => {
+      return doc.id === id;
+    });
+
+    const total = itemId.total_purchases + 1;
+    const previousEstimate = itemId.previous_estimate;
+    const lastPurchased = itemId.last_purchased_date;
+
+    const today = Date.now();
+
+    const daysSincePurchase = today - lastPurchased;
+    //msToTime function below
+    const milliseconds = Math.round(msToTime(daysSincePurchase));
```

Comment:
The milliseconds variable appears to be set to the milliseconds converted to minutes/hours/etc using msToTime. It looks like the calculateEstimate function expects it formatted in the number of days (when I peeked in the node module)? For example, if msToTime returns 5, for 5 minutes, I think the function will think that it represents 5 days. What do you think? Maybe if it's less than a whole day, we can say 1 day?